climada/engine/forecast.py

```diff
@@ -1079,18 +1079,6 @@ def _plot_warn(
             polygon_file_crs, self._impact[haz_ind].crs, always_xy=True
         )
         # checking the decision dict and define the corresponding functions
-        if not (
-            isinstance(decision_dict["probability_aggregation"], float)
-            & isinstance(decision_dict["area_aggregation"], float)
-        ):
-            ValueError(
-                " If decision_level is 'exposure_point',"
-                + "parameters probability_aggregation and "
-                + "area_aggregation of "
-                + "Forecast.plot_warn_map() must both be "
-                + "floats between [0..1]. Which each "
-                + "specify quantiles."
-            )
```

Comment:
How is this related to the Exposures tag?

Reply:
Forecast will be overhauled anyway. I thinks it's best not to touch that class now. Can you revert the changes, even though the error clearly should be raised? 😅